app/services/importer/scraper/workday_cookie_fetcher.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Importer
+  module Scraper
+    class WorkdayCookieFetcher < ApplicationTask
+      include Capybara::DSL
+
+      def initialize(details)
+        @create_account_url = details[:create_account_url]
+        @sign_in_url = details[:sign_in_url]
+        @target = details[:target_url]
+        Capybara.configure do |config|
```

Comment:
Questioning config in the initialise method. @patanj101 can you also provide a view

Reply:
Since it doesn't affect processable ... I would recommend to put all the logic behind @session (and thus using_session(@session)) as the first step of the process ...
The last would be quit

Reply:
Sorry, I'm not sure I understand either comment here. Where else would config go? It's specific to this class, not a general configuration.

Also, I'm not sure whether or not I need to handle config before the new session is initialized. Can't find information in the docs about this, but ChatGPT says yes.

Reply:
![Screenshot 2024-08-06 at 11 42 07](https://github.com/user-attachments/assets/c25ee792-6385-47c9-951b-851c54959da3)
Apologies, I guess I was being dense. Is this what you're suggesting @patanj101?